otherlibs/dyn/dyn.ml

```diff
@@ -112,7 +112,11 @@ let rec pp =
   | Variant (v, xs) ->
     Pp.hvbox
       ~indent:2
-      (Pp.concat [ Pp.verbatim v; Pp.space; Pp.concat_map ~sep:(Pp.char ',') xs ~f:pp ])
+      (Pp.concat
+         [ Pp.verbatim v
+         ; Pp.space
+         ; Pp.concat_map ~sep:(Pp.seq (Pp.char ',') Pp.space) xs ~f:pp
```

Comment:
Shouldn't it be `Pp.seq Pp.space (Pp.char ',')`? We use a leading `,` and `;` everywhere in our code.

Reply:
That works too, I was being consistent with what was being done for Tuple.

Reply:
Okay, then leave it. We can change the style in subsequent PR's.

Reply:
We have a choice here.
```
Pp.custom_break ~fits:(",", 1, "") ~breaks:("", 0, ", ")
```
Gives:
```
a, b, c
```
when fitting on a line and
```
a
, b
, c
```
when breaking.

We can reduce the space in ~fits to 0 to preserve the tight constructor tuples we already had but I think it looks clearer with a space.

I can also make the same break modification for tuples.

Reply:
I'll leave it for later however.